taskcluster/ci/complete/kind.yml

```diff
@@ -27,7 +27,7 @@ task-defaults:
     run:
         command:
             task-reference:
-                /builds/worker/checkouts/vcs/taskcluster/scripts/are_dependencies_completed.py {dependencies}
+                /builds/worker/checkouts/vcs/taskcluster/scripts/are_dependencies_completed.py <self>
```

Comment:
TIL! This variable is not set in the task definition. I guess it's made available by the worker. If so, I'd prefer to make things explicit rather than implicit.